.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "trailingComma": true
+}
```

Comment:
Please ignore prettier or every fmt config file, not everyone uses these things.